src/library/features/analysis/analysisfeature.cpp

```diff
@@ -54,62 +52,74 @@ QVariant AnalysisFeature::title() {
     return m_Title;
 }
 
-QIcon AnalysisFeature::getIcon() {
-    return QIcon(":/images/library/ic_library_prepare.png");
+QString AnalysisFeature::getIconPath() {
+    return ":/images/library/ic_library_prepare.png";
 }
 
-void AnalysisFeature::bindWidget(WLibrary* libraryWidget,
-                                 KeyboardEventFilter* keyboard) {
-    m_pAnalysisView = new DlgAnalysis(libraryWidget,
-                                      m_pConfig,
-                                      m_pTrackCollection);
-    connect(m_pAnalysisView, SIGNAL(loadTrack(TrackPointer)),
-            this, SIGNAL(loadTrack(TrackPointer)));
-    connect(m_pAnalysisView, SIGNAL(loadTrackToPlayer(TrackPointer, QString)),
-            this, SIGNAL(loadTrackToPlayer(TrackPointer, QString)));
-    connect(m_pAnalysisView, SIGNAL(analyzeTracks(QList<TrackId>)),
-            this, SLOT(analyzeTracks(QList<TrackId>)));
-    connect(m_pAnalysisView, SIGNAL(stopAnalysis()),
-            this, SLOT(stopAnalysis()));
+QString AnalysisFeature::getSettingsName() const {
+    return "AnalysisFeature";
+}
 
-    connect(m_pAnalysisView, SIGNAL(trackSelected(TrackPointer)),
-            this, SIGNAL(trackSelected(TrackPointer)));
+QWidget* AnalysisFeature::createPaneWidget(KeyboardEventFilter*, int paneId) {
+    WTrackTableView* pTable = createTableWidget(paneId);
```

Comment:
The `createPaneWidget` function is assumed to return a `QWidget` without parent, the parent is assigned in the `LibraryPaneManager` right after the function is called. Previously the parent was passed as a parameter but I removed it and I don't remember the reason :confused: